### PR TITLE
Update HP

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
hp

**Issue:** contributes to #9072 

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from https://brandcentral.hp.com/us/en/elements/hp-logo.html. Please review with the logo here. I found three different versions on their own website itself.
Color: Unchanged

### CODE REVIEW

File: icons/hp.svg
Comment (adamrusted) [inline icons/hp.svg:1]: I'm spying a slight difference when comparing to `HP_logo_black.svg`.
image

Could you take another look please @PratyushJayachandran? 🙏

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/hp.svg
Comment (PratyushJayachandran) [reply icons/hp.svg:1]: Seems like a precision issue. I increased the precision to 4.

Comment (PratyushJayachandran) [reply icons/hp.svg:1]: Is there still an issue @adamrusted?

Comment (adamrusted) [reply icons/hp.svg:1]: Not looked at it yet, bear with.

**Review by adamrusted — APPROVED** [review]

**Pull request merged**